### ISSUE TITLE
Release 0.0.7

### ISSUE THREAD

**Issue by XavierCLL** [gee-community/qgis-earthengine-plugin#159]
IMO there are some important things we need to fix before the next release:

* I did some code comments in the PR maybe you didn't see it https://github.com/gee-community/qgis-earthengine-plugin/pull/156
* Issue 1 in that PR is a critical bug IMO
* Include in the release https://github.com/gee-community/qgis-earthengine-plugin/pull/154

**Comment by XavierCLL** [comment]
@gena Most of these things are easy to do, It's not a good idea to release it in a hurry.

**Comment by gena** [comment]
Thanks, I saw comments and added some checks, as well as a few commits to fix more things / bugs.  

I've added a comment in that PR and did not see reaction. These contrib things should be handled differently, maybe even move to some other plugin to be updated more frequently. Excluded for now.

It's released on GitHub and Uploaded to QGIS Plugins Repo. I have to update the warning in another repo to ask people to upgrade to 0.0.7, will do it later in the coming days.

Thanks @XavierCLL and @SiggyF for your contribution with reviews and bug fixes in this version.
